### PR TITLE
Update Triad and Parthenon Collaboration copyright to 2021

### PR DESCRIPTION
**Pull request by AndrewGaspar**
## PR Summary

Title.

## PR Checklist

- [x] Code passes cpplint
- [x] New features are documented.
- [x] Adds a test for any bugs fixed. Adds tests for new features.
- [x] Code is formatted
- [x] Changes are summarized in CHANGELOG.md

### CODE REVIEW

**Review by pgrete — COMMENTED** [review]
I'm not really familiar with the technical details of copyrighting so I wonder whether 2020-2021 would be more appropriate?

**Comment by AndrewGaspar** [comment]
> I'm not really familiar with the technical details of copyrighting so I wonder whether 2020-2021 would be more appropriate?

You and me both! @gshipman ?

**Review by JoshuaSBrown — APPROVED** [review]

File: src/config.hpp.in [hunk 1]
```diff
@@ -21,7 +21,7 @@
 //  config.hpp file will be created (overwriting the last) from this template. This new
 //  file contains Athena++ specific cpp macros and definitions set by configure.
 
-#define COORDINATE_TYPE @COORDINATE_TYPE@
+#define COORDINATE_TYPE @COORDINATE_TYPE @
```
Comment (AndrewGaspar) [inline src/config.hpp.in]: wtf, i don't know how this happened.

Comment (JoshuaSBrown) [reply src/config.hpp.in]: The formatter will do this if it runs on this file.

**Pull request closed without merging**